packages/pixeloven-core/common/package.json

```diff
@@ -17,29 +17,32 @@
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
+    "@types/node": "10.17.12"
+  },
   "devDependencies": {
-    "@pixeloven/tasks": "^6.0.0-beta.22",
+    "@pixeloven-tooling/compiler": "^6.0.0-beta.22",
+    "@pixeloven-tooling/linter": "^6.0.0-beta.22",
+    "@pixeloven-tooling/pretty": "^6.0.0-beta.22",
+    "@pixeloven-tooling/test": "^6.0.0-beta.22",
     "lint-staged": "9.5.0"
   },
   "scripts": {
-    "clean": "pixeloven-tasks clean",
```

Comment:
Is there no need for clean or document any more?

Reply:
We do incremental builds now and I never got around to implementing the typedocs stuff. I'm going to support the typedocs stuff for the main CLI for now. Eventually plan on moving it as an addon.